browser/ui/views/tabs/brave_tab_strip.cc

```diff
@@ -65,7 +65,11 @@ bool BraveTabStrip::IsVerticalTabsFloating() const {
   auto* vertical_region_view =
       browser_view->vertical_tab_strip_widget_delegate_view()
           ->vertical_tab_strip_region_view();
-  DCHECK(vertical_region_view);
+  if (!vertical_region_view) {
+    // Could be null when we end up here from VerticalTabStripWidgetDelegateView
```

Comment:
when/why would it be null? Please add to the comment

Reply:
Unfortunately, I didn't note which tests were failing and now I reverted the change and the tests are all fine, so I guess we don't need this change.